static/src/javascripts/projects/common/modules/commercial/acquisitions-copy.js

```diff
@@ -23,6 +30,13 @@ const ctaLinkSentence = (
 ): string =>
     `<span class="contributions__highlight"> For as little as ${currencySymbol}1, you can support the Guardian – and it only takes a minute.</span> <a href="${supportUrl}" target="_blank" class="u-underline">Make a contribution</a>`;
 
+const ctaLinkSentenceWorldCupLiveblogPlayful = (
+    supportUrl: string,
+    contributionUrl: string,
```

Comment:
I guess we can delete `contributionUrl` from these and from the copy functions

Reply:
Great spot